src/site/content/en/blog/redbus-inp/index.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: How redBus improved their website's Interaction to Next Paint (INP) and increased sales by 7%
+subhead: |
+  INP optimizations helped redBus increase sales by ~7%
```

Comment:
style guide doesn't like ~, can we use a word here?

Reply:
Yeah, I'll replace it with "about" and push a new change right away.